civictechprojects/templates/project_signup.html

```diff
@@ -69,32 +69,32 @@ <h2 class="form-group subheader">DETAILS</h2>
 
             <div class="form-group">
                 <label for="brief_description">Describe This Project</label>
-                <textarea class="form-control" id="brief_description" placeholder="This will appear as project introduction" rows=3></textarea>
+                <textarea class="form-control" name="brief_description" placeholder="This will appear as project introduction" rows=3></textarea>
```

Comment:
This wasn't added by your change, but I noticed it during testing.  Name should be "project_description" in order for it to be saved by the view.